app/includes/layoutPrint.njk

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="ie8" lang="en"><![endif]-->
+<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
+<!--[if gt IE 9]><!--><html lang="en"><!--<![endif]-->
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="{% if pageDescription %}{{ pageDescription }}{% else %}NHSD Order Form{% endif %}">
+
+    <title>{% if title %}{{ title }}{% else %}Buying Catalogue - Order Form{% endif %}</title>
+
+    <link rel="stylesheet" href="{{ config.baseUrl }}/css/main.min.css">
+
+    <script src="{{ config.baseUrl }}/js/main.bundle.min.js" defer></script>
+
+    <link href="https://www.nhs.uk/" rel="preconnect">
+    <link href="https://assets.nhs.uk/" rel="preconnect" crossorigin>
+    <link type="font/woff2" href="https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.woff2" rel="preload" as="font" crossorigin>
+    <link type="font/woff2" href="https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.woff2" rel="preload" as="font" crossorigin>
+
+    <link rel="shortcut icon" href="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/favicon.ico" type="image/x-icon">
+    <link rel="apple-touch-icon" href="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/apple-touch-icon-180x180.png">
+    <link rel="mask-icon" href="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/favicon.svg" color="#005eb8">
+    <link rel="icon" sizes="192x192" href="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/favicon-192x192.png">
+    <meta name="msapplication-TileImage" content="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/mediumtile-144x144.png">
+    <meta name="msapplication-TileColor" content="#005eb8">
+    <meta name="msapplication-square70x70logo" content="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/smalltile-70x70.png">
+    <meta name="msapplication-square150x150logo" content="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/mediumtile-150x150.png">
+    <meta name="msapplication-wide310x150logo" content="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/widetile-310x150.png">
+    <meta name="msapplication-square310x310logo" content="{{ config.baseUrl }}/nhsuk-frontend/assets/favicons/largetile-310x310.png">
+    <meta name="robots" content="noindex">
+    <meta name="googlebot" content="noindex">
+  </head>
+
+  <body onload="window.print();">
```

Comment:
If the user has disabled javascript will the window.print() still trigger? Not sure we need to cater for this as it stands unless there is a business requirement.

Reply:
They'll have to do the print page manually from the browser

Reply:
As someone who tends to have JS disabled by default (beloved by Josh), websites automatically popping up print windows is something of a bugbear of mine (assuming I have JS enabled for that particular site).

Reply:
Oh it's guys like you causing us problems 😂  yeah i'm not a fan of javascript running when a page loads regardless. But this is what the BAs wanted to make it easier for the end user. Whom may not be aware of how to print a webpage and also save a few clicks.